addons/phone_validation/tests/test_phonenumbers_patch.py

```diff
@@ -0,0 +1,28 @@
+from odoo.tests.common import TransactionCase
+from odoo.tests import tagged
+from odoo.tools.parse_version import parse_version
+import odoo.addons.phone_validation.lib.phonenumbers_patch
+
+@tagged('post_install', '-at_install')
+class TestPhonenumbersPatch(TransactionCase):
+    @classmethod

```

Comment:
Why a class method specifically ?

Reply:
Honestly, I haven't done much testing before. But I saw this in the python docs: `setUpClass` is called with the class as the only argument and must be decorated as a `classmethod()` : https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUpClass

Reply:
in this is necessary since we have a class but not an object, but not here since the methods are called from an object (this doesn't cause an issue but not necessary)

```py
>>> class Test():
...     @classmethod
...     def withclass(cls):
...         print(cls)
...     def withnoclass(self):
...         print(self)
... 
>>> Test().withclass()
<class '__main__.Test'>
>>> Test.withclass()
<class '__main__.Test'>
>>> Test().withnoclass()
<__main__.Test object at 0x7fcf77b58e20>
>>> Test.withnoclass()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: Test.withnoclass() missing 1 required positional argument: 'self'
```